lighthouse-core/config/default-config.js

```diff
@@ -421,12 +421,12 @@ const defaultConfig = {
     'performance': {
       title: str_(UIStrings.performanceCategoryTitle),
       auditRefs: [
-        {id: 'first-contentful-paint', weight: 15, group: 'metrics', acronym: 'FCP', relevantAudits: m2a.fcpRelevantAudits},
-        {id: 'speed-index', weight: 15, group: 'metrics', acronym: 'SI'},
+        {id: 'first-contentful-paint', weight: 10, group: 'metrics', acronym: 'FCP', relevantAudits: m2a.fcpRelevantAudits},
```

Comment:
I totally missed this `m2a` business, sounds like someone has some apologizing to do to `atob`/`btoa` ;)